proto/google/api/annotations.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015, Google Inc.
```

Comment:
Good to know. Right now there isn't a great story around packaging, but we are working around it by pushing ourselves (I'm hoping that eventually this is done via googleapis). However, I can work today on getting the protos on npm.